docs/dashboards.md

```diff
@@ -164,7 +168,7 @@ With Istio, this is done automatically by the Envoy sidecar.
 1. Run the following command:
 
     ```{.shell .language-shell}
-    kubectl exec svc/customers -- curl -s localhost:15020/stats/prometheus \
+    kubectl exec svc/web-frontend -- wget -qO - localhost:15020/stats/prometheus \
```

Comment:
i think what happened here is that recently a user pointed out that the `customers` and `web-frontend` images had a lot of cve's (on account of having been built a long time ago).  i recently updated the images to eliminate most/all of the cve's; that included using a different base image, likely one that didn't inherit curl.